backend/corpora/common/corpora_orm.py

```diff
@@ -456,13 +456,12 @@ class DbGeneset(Base, AuditMixin):
     description = Column(String)
     genes = Column(JSONB)
     collection_id = Column(String, nullable=False)
-    collection_visibility = Column(Enum(CollectionVisibility), nullable=False)
     collection = relationship("DbCollection", uselist=False, back_populates="genesets")
     datasets = relationship("DbDataset", secondary="geneset_dataset_link", back_populates="genesets")
 
     __table_args__ = (
-        ForeignKeyConstraint([collection_id, collection_visibility], [DbCollection.id, DbCollection.visibility]),
-        UniqueConstraint("name", "collection_id", "collection_visibility", name="_geneset_name__collection_uc"),
+        ForeignKeyConstraint((collection_id,), (DbCollection.id,)),
```

Comment:
```suggestion
```

Reply:
This would change the expected behavior from before. Could you explain why you want this removed?

Reply:
this changes goes with https://github.com/chanzuckerberg/single-cell-data-portal/pull/2061#discussion_r821191412

Reply:
You can put this inline with `revision_of`

Reply:
ah got it, with the comment above 👍